test/controllers/image_controller_test.rb

```diff
@@ -693,6 +997,30 @@ def test_add_images_strip_gps
     assert_true(img.gps_stripped)
   end
 
+  def test_add_images_process_image_fail
+    login("rolf")
+    obs = observations(:coprinus_comatus_obs)
+    setup_image_dirs
+    fixture = "#{MO.root}/test/images/geotagged.jpg"
+    fixture = Rack::Test::UploadedFile.new(fixture, "image/jpeg")
+    Image.any_instance.stubs(:process_image).returns(false)
```

Comment:
@pellaea: Is there an image (or pseudo-image) file that will make `Image#process_image` fail?